src/Mod/PartDesign/Gui/TaskHoleParameters.cpp

```diff
@@ -140,20 +139,25 @@ TaskHoleParameters::TaskHoleParameters(ViewProviderHole *HoleView, QWidget *pare
     if (pcHole->HoleCutType.isValid())
         holeCutType = pcHole->HoleCutType.getValueAsString();
 
+    else if (holeCutType == "None") {

```

Comment:
Is this "else" here intended?

Reply:
oops, this one gets an "if" the Counterbore an "else if" -> fix is on the way

Reply:
Oops, this is now fixed.